go.mod

```diff
@@ -3,19 +3,27 @@ module github.com/codeready-toolchain/registration-service
 go 1.12
 
 require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/dnaeon/go-vcr v1.0.1 // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/fabric8-services/fabric8-auth-client v0.0.0-20190313133658-59d93fb7492b
+	github.com/fabric8-services/fabric8-common v0.0.0-20190529115613-c08218d5adf2
 	github.com/gin-contrib/gzip v0.0.1
 	github.com/gin-contrib/sse v0.1.0 // indirect
 	github.com/gin-gonic/gin v1.4.0
+	github.com/goadesign/goa v2.0.5+incompatible
```

Comment:
I thought we didn't use goa?

Reply:
We don't. I am looking at why this is not updating on build.

Reply:
Fixed 31a3d4a.